airflow/__main__.py

```diff
@@ -34,6 +35,15 @@ def main():
         os.environ['KRB5CCNAME'] = conf.get('kerberos', 'ccache')
         os.environ['KRB5_KTNAME'] = conf.get('kerberos', 'keytab')
 
+    # if dags folder has to be set to configured value, make sure it is set properly (needed on Dask-Workers)
```

Comment:
This is probably not an ideal solution and I forgot to put this in from our repository in the first commit. On Windows the Dask-Worker tries to get the DAG at the location where it is stored on the Airflow server (e.g. /home/airflow/dags). So here we actually force it to use the DAG path in the config file... I don't know how this is handled on a setup involving only Linux machines.

Reply:
This is not the right fix for this.

There's a bug in a previous feature where the "dag folder" should be replaced with `DAG_FOLDER` which is then automaticall replaced, but this isn't working.

We should fix that rather than adding a new config and new way of making this

Reply:
Yep. Also see  https://github.com/apache/airflow/discussions/16423 - might be the right time to fix it.

Reply:
Windows API can handle both forward and backward slashes (except some very old legacy stuff, none of which is used by Python’s path libraries), so it should be fine if we use forward slashes everywhere. For rare cases where some third-party packages rely on legacy APIs, we can call `os.path.normpath` to convert the slashes.

Reply:
Ah cool. Did not know that (not a Windows user :D) 

 I thought it's a Java-specfic behaviour only (it was part of the specification from the beginning). So no problem here.